src/test/java/com/glovoapp/planningpoker/e2e/logintosession/JoinOtherUserSessionTest.java

```diff
@@ -0,0 +1,40 @@
+package com.glovoapp.planningpoker.e2e.logintosession;
+
+import com.glovoapp.planningpoker.e2e.WebDriverFactory;
+import com.glovoapp.planningpoker.e2e.pages.MainPage;
+import com.glovoapp.planningpoker.e2e.pages.SessionPage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+
+import static com.glovoapp.planningpoker.e2e.WebDriverFactory.getDriver;
+
+class JoinOtherUserSessionTest {
+    private static final String sessionName = "test1";
+
+    private final WebDriver driver = getDriver();
+    private final WebDriver driver2 = getDriver();
```

Comment:
Please don't use digits in variable names:

```suggestion
    private final WebDriver secondDriver = getDriver();
```